css/nav.css

```diff
@@ -200,7 +230,7 @@ section#credits {
 
     nav a:not(#nav-home) {
         padding: 0.55rem 2.7rem;
-        transform-origin: 50% 100%;
+        transform-origin: 50% 0%;
```

Comment:
(highlighted CSS unrelated to comment) while you're here, would you mind polishing our buttons a bit:
 - the 'open pack' button doesn't change colors on hover
 - the hover color doesn't appear if you are keyboard focused on the button